packages/diagrams/frontend/sirius-components-diagrams/src/renderer/DiagramRenderer.tsx

```diff
@@ -72,11 +75,14 @@ import { useDiagramSelection } from './selection/useDiagramSelection';
 import { useShiftSelection } from './selection/useShiftSelection';
 import { useSnapToGrid } from './snap-to-grid/useSnapToGrid';
 
-import 'reactflow/dist/style.css';
-
 const GRID_STEP: number = 10;
 
-export const DiagramRenderer = memo(({ diagramRefreshedEventPayload }: DiagramRendererProps) => {
+export const DiagramRenderer = memo(() => {
```

Comment:
I used a context to store diagramRefreshedEventPayload because we have in diagramRepresentation
`      <DiagramContext.Provider
        value={{
          editingContextId,
          diagramId: representationId,
          refreshEventPayloadId: state.diagramRefreshedEventPayload.id,
          readOnly,
        }}>
`
And the `refreshEventPayloadId` is then used in various hooks like `useAdjustSize`